app/models/manageiq/providers/infra_manager.rb

```diff
@@ -44,5 +44,24 @@ def self.merge_config_settings(cfg = VMDB::Config.new("vmdb"))
       path = [:ems, :ems_redhat, :service, :read_timeout]
       cfg.merge_from_template_if_missing(*path)
     end
+
+    def self.credential_validation(ems, auth_type, creds, revalidate)
+      ems.update_authentication(creds, :save => false)
```

Comment:
I thought that `:save` should use `revalidate` here and not default to `false`, but maybe this is something that @jrafanie changed when he updated the way we handle revalidating the authentication records...  @jrafanie?